topic/multi-model/multi-model-offshore-wind-farms.ipynb

```diff
@@ -666,20 +723,24 @@
    ],
    "source": [
     "query = \"\"\"\n",
-    "SELECT \n",
-    "    ts AS hour_of_day, \n",
-    "    output AS hour_output, \n",
-    "    sum(output) OVER (ORDER BY ts ASC) AS output_day_so_far\n",
+    "SELECT DISTINCT\n",
```

Comment:
GROUP BY maybe would be faster? probably does not make a difference in this small dataset